app/components/TeamLogo.ts

```diff
@@ -4,8 +4,20 @@ import { Avatar } from "./Avatar";
 
 const TeamLogo = styled(Avatar)`
   border-radius: 4px;
-  box-shadow: inset 0 0 0 1px ${s("divider")};
   border: 0;
+  position: relative;
+
+  &::before {
```

Comment:
Just a random styling fix I noticed in the menus here – avoiding a grey outline around all team logos